bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/RuleRegistryImpl.java

```diff
@@ -32,10 +38,14 @@
     private Set<String> disabledRuledSet = new HashSet(0);
     private Logger logger;
     private Storage<Boolean> disabledRulesStorage;
+    private EventPublisher eventPublisher;
```

Comment:
The base class AbstractRegistry already holds a eventPublisher variable. Use this one.